styx-common/src/main/java/com/spotify/styx/api/Api.java

```diff
@@ -38,7 +38,7 @@
 public final class Api {
 
   public enum Version {
-    V0, V1, V2, V3;
+    V3;
 
     public String prefix() {
       return "/api/v" + ordinal();
```

Comment:
LOL, didn't see this at first. Would be fun, but also maybe we should use the string `"v3"` in some places rather than this thing.

Reply:
Get name and then lower case instead of ordinal should be ok.

Reply:
cool that we found this at the same time 😄